packages/e2e-tests/specs/editor/plugins/iframed-multiple-block-stylesheets.test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	createNewPost,
+	deactivatePlugin,
+	insertBlock,
+	getEditedPostContent,
+	openDocumentSettingsSidebar,
+	clickButton,
+	canvas,
+} from '@wordpress/e2e-test-utils';
+
+async function getComputedStyle( context, property ) {
+	await context.waitForSelector(
+		'.wp-block-test-iframed-multiple-stylesheets'
+	);
+	return await context.evaluate( ( prop ) => {
+		const container = document.querySelector(
+			'.wp-block-test-iframed-multiple-stylesheets'
+		);
+		return window.getComputedStyle( container )[ prop ];
+	}, property );
+}
+
+describe( 'iframed multiple block stylesheets', () => {
+	beforeEach( async () => {
+		await activatePlugin( 'gutenberg-test-iframed-multiple-stylesheets' );
+		await createNewPost( { postType: 'page' } );
+	} );
+
+	afterEach( async () => {
+		await deactivatePlugin( 'gutenberg-test-iframed-multiple-stylesheets' );
+	} );
+
+	it( 'should load multiple block stylesheets in iframe', async () => {
+		await insertBlock( 'Iframed Multiple Stylesheets' );
+
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+		await openDocumentSettingsSidebar();
+		await clickButton( 'Page' );
+		await clickButton( 'Template' );
+		await clickButton( 'New' );
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.type( 'Iframed Test' );
+		await clickButton( 'Create' );
+		await page.waitForSelector( 'iframe[name="editor-canvas"]' );
+
+		// Style loaded from the main stylesheet.
+		expect( await getComputedStyle( canvas(), 'border-style' ) ).toBe(
+			'dashed'
+		);
+
+		// Style loaded from the additional stylesheet.
+		expect( await getComputedStyle( canvas(), 'border-color' ) ).toBe(
+			'rgb(255, 0, 0)'
+		);
+
+		expect( console ).toHaveErrored();
```

Comment:
What's the reason that `console` errors here? It means that the errors is triggered in the test code, not in the application.

Reply:
The console throws this error: `Stylesheet iframed-multiple-stylesheets-style2-css was not properly added.
    For blocks, use the block API's style (https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#style) or editorStyle (https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#editor-style).
    For themes, use add_editor_style (https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-support/#editor-styles). <link rel=\"stylesheet\" id=\"iframed-multiple-stylesheets-style2-css\" href=\"http://localhost:8889/wp-content/plugins/gutenberg-test-plugins/iframed-multiple-stylesheets/style2.css?ver=1633432626\" media=\"all\">`
It's the same error as here: https://github.com/WordPress/gutenberg/blob/99096e1693444286096b15a703555eff4fc73583/packages/e2e-tests/specs/editor/plugins/iframed-inline-styles.test.js#L61-L65

Reply:
These errors were added in https://github.com/WordPress/gutenberg/pull/31870 and we've been ignoring them. 
They cover 2 cases: Adding styles when adding a new block, and adding styles from a theme. However, when adding a style from a plugin or adding an additional stylesheet to a block, the editor console error appears. 
Plugins can't use the `add_editor_style` because as stated in the [function docs](https://developer.wordpress.org/reference/functions/add_editor_style/):
> The parameter $stylesheet is the name of the stylesheet, relative to the theme root. It also accepts an array of stylesheets. It is optional and defaults to ‘editor-style.css’.

It only accepts stylesheet(s) **relative to the theme root** and therefore can't be used outside a theme.

Reply:
@gziolo Since this is a known issue and not related to this PR, I assume it's not a blocker?

Reply:
If it's a known issue that is tracked already then we can leave it for later 👍🏻 

Can you add a link to the issue near the line which suppresses the error?